tools/db-compiler-lite/src/main/java/com/grab/databinding/stub/rclass/generator/ResToRClassGenerator.kt

```diff
@@ -73,7 +73,7 @@ class ResToRClassGeneratorImpl constructor(
                         .toTypedArray()
 
                     fields.add(
-                        FieldSpec.builder(type, it.name)
+                        FieldSpec.builder(type, it.name.replace(":", "_"))
```

Comment:
Could you please move this logic to https://github.com/grab/grab-bazel-common/blob/240a1c393180c5121032e82e4603a124544c241b/tools/db-compiler-lite/src/main/java/com/grab/databinding/stub/rclass/parser/ResToRParser.kt#L223? I think that is where the `:` is getting added. Would prefer to have less processing done on generator class and fix things on the parser side.